src/IssueViz.Security/IssueVizDisplayPositionCalculator.cs

```diff
@@ -0,0 +1,63 @@
+﻿/*
+ * SonarLint for Visual Studio
+ * Copyright (C) 2016-2020 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+using System;
+using SonarLint.VisualStudio.IssueVisualization.Models;
+
+namespace SonarLint.VisualStudio.IssueVisualization.Security
+{
+    internal interface IIssueVizDisplayPositionCalculator
+    {
+        int GetLine(IAnalysisIssueVisualization issueViz);
+
+        int GetColumn(IAnalysisIssueVisualization issueViz);
+    }
+
+    internal class IssueVizDisplayPositionCalculator : IIssueVizDisplayPositionCalculator
+    {
+        int IIssueVizDisplayPositionCalculator.GetColumn(IAnalysisIssueVisualization issueViz)
+        {
+            int zeroBasedColumn;
+
+            if (!CanUseSpan(issueViz))
+            {
+                zeroBasedColumn = issueViz.Issue.StartLineOffset;
+            }
+            else
+            {
+                var position = issueViz.Span.Value.Start;
+                var line = position.GetContainingLine();
+                zeroBasedColumn = position.Position - line.Start.Position;
+            }
+
+            // both SQ hotspot column and VS column are zero-based
```

Comment:
Comment is now out of date (says "hotpost").
Are we sure that this is the case for regular issues as well? (I'd assume that yes but it's worth to double check)

Reply:
I checked, they are (it would be very surprising if they weren't, given that hotspots were originally treated as normal issues).